libraries/AP_HAL_SITL/UARTDriver.cpp

```diff
@@ -152,7 +153,19 @@ size_t UARTDriver::write(const uint8_t *buffer, size_t size)
     if (size <= 0) {
         return 0;
     }
-    _writebuffer.write(buffer, size);
+    if (_unbuffered_writes) {
+        struct timeval tp;
+        gettimeofday(&tp, nullptr);
+        uint64_t time = 1.0e6 * (tp.tv_sec + (tp.tv_usec * 1.0e-6));
+        DataFlash_Class::instance()->Log_Write("SBUS", "TimeUS,ToD", "QQ",
+                                               AP_HAL::micros64(), time);
+        // write buffer straight to the file descriptor
+        ::write(_fd, buffer, size);
```

Comment:
write

Reply:
isn't it better to be explicit? Or is that a meaningless concept in C++? (I usually have no idea what namespace is active)